scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatWriter.scala

```diff
@@ -71,6 +71,12 @@ class FormatWriter(formatOps: FormatOps) {
           sb.append(rewrittenToken)
       }
 
+      location.optionalBraces.toSeq.sortBy(-_._1).foreach {
```

Comment:
```suggestion
      location.optionalBraces.toSeq.sortBy{case (indent, _) => -indent}.foreach {
```
it could be a bit clearer when reading.

Reply:
done